dividers/src/main/java/com/decathlon/vitamin/compose/dividers/VitaminDividers.kt

```diff
@@ -0,0 +1,85 @@
+package com.decathlon.vitamin.compose.dividers
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.decathlon.vitamin.compose.foundation.VitaminTheme
+
+object VitaminDividers {
+
+    /**
+     * Full-bleed dividers separate content into sections. They can also be used to establish a hierarchy
+     * @param modifier The `Modifier` to be applied to this dividers
```

Comment:
With kdoc, a link to a class reference is written `[Modifier]`.
Change it here and in all functions too in this file.